rust/devlog.md

```diff
@@ -0,0 +1 @@
+# Devlog
```

Comment:
Drop a link in here to the 100,000'th issue on rust that the emoji choice came from!

Reply:
Added the link.